test/configuration/displayname.spec.conf/displayname_schema.json

```diff
@@ -0,0 +1,780 @@
+            {
+              "table_name": "accommodation",
+              "schema_name": "displayname_schema",
+              "column_name": "thumbnail"
+            }
+          ],
+          "annotations": {
+            "comment": ["thumbnail"]
+          },
+          "referenced_columns": [
+            {
+              "table_name": "file",
+              "schema_name": "displayname_schema",
+              "column_name": "id"
+            }
+          ]
+        },
+        {
+          "comment": null,
+          "foreign_key_columns": [
+            {
+              "table_name": "accommodation",
+              "schema_name": "displayname_schema",
+              "column_name": "cover"
+            }
+          ],
+          "annotations": {
+            "comment": ["thumbnail"]
+          },
+          "referenced_columns": [
+            {
+              "table_name": "file",
+              "schema_name": "displayname_schema",
+              "column_name": "id"
+            }
+          ]
+        }
+      ],
+      "table_name": "accommodation",
+      "schema_name": "displayname_schema",
+      "column_definitions": [
+        {
+          "comment": null,
+          "name": "id",
+          "default": null,
+          "nullok": false,
+          "type": {
+            "typename": "int4"
+          },
+          "annotations": {
+            "comment": [
+              "hidden"
+            ]
+          }
+        },
+        {
+          "comment": null,
+          "name": "title",
+          "default": null,
+          "nullok": false,
+          "type": {
+            "typename": "text"
+          },
+          "annotations": {
+            "comment": [
+              "title",
+              "top",
+              "orderby"
+            ],
+            "description": {
+              "display": "Name of Accommodation"
+            },
+            "facetOrder": [
+              "1"
+            ]
+          }
+        },
+        {
+          "comment": null,
+          "name": "website",
+          "default": null,
+          "nullok": true,
+          "type": {
+            "typename": "text"
+          },
+          "annotations": {
+            "comment": [
+              "url"
+            ],
+            "description": {
+              "display": "Website"
+            }
+          }
+        },
+        {
+          "comment": null,
+          "name": "category",
+          "default": null,
+          "nullok": false,
+          "type": {
+            "typename": "text"
+          },
+          "annotations": {
+            "comment": ["top"],
+            "description": {
+              "display": "Category"
+            },
+            "facetOrder": [
+              "2"
+            ]
+          }
+        },
+        {
+          "comment": null,
+          "name": "rating",
+          "default": 3,
+          "nullok": false,
+          "type": {
+            "typename": "float4"
+          },
+          "annotations": {
```

Comment:
I copy/pasted the schema that has been created before and just added the `tag:misd.isi.edu,2015:display` annotation for some specific tables. I'll get rid of this unused annotations.